rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -327,6 +327,7 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 			"\nKey I: Show RSX method detail."
 			"\nKey F10: Perform step-over on instructions. (skip function calls)"
 			"\nKey F11: Perform single-stepping on instructions."
+			"\nKey F12: Show the SPU disassmebler dialog."
```

Comment:
Typo. In disassembler.
Also, this doesn't match the actual key combo

Reply:
I know, its a draft.